### PR TITLE
fixes: small but a few

### PR DESCRIPTION
**Pull request by mockdot**
## Description

- Fixed financing time discounted incorrectly on playerDropped
- Fixed certain users not being brought back after test drive
- Reverted notifies to Functions.Notify
- Adjusted buying/test drive spawns so they don't collide with players parking at the vehicle shop

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -72,7 +71,7 @@ end
 
 --- Gets the owned vehicles based on financing then opens a menu
 local function showFinancedVehiclesMenu()
-    local vehicles = lib.callback.await('qb-vehicleshop:server:getVehicles')
+    local vehicles = lib.callback.await('qb-vehicleshop:server:getVehicles', false)
```
Comment (Manason) [inline client/main.lua]: Is the explicit false needed? I would have expected it to default if not set explicitly.

File: client/main.lua [hunk 2]
```diff
@@ -419,19 +414,19 @@ end
...
+
     CreateThread(function()
         while InTestDrive do
-            if GetGameTimer() < gameTimer + tonumber(1000 * time) then
-                local secondsLeft = GetGameTimer() - gameTimer
-                if secondsLeft >= tonumber(1000 * time) - 20 then
+            local currentGameTime = GetGameTimer()
+            local secondsLeft = currentGameTime - gameTimer
+            if currentGameTime < gameTimer + timeMs then
+                if secondsLeft >= timeMs - 50 then
```
Comment (Manason) [inline client/main.lua:424]: Why the change from -20 to -50? Also did you end up trying the condition where secondsLeft <= 0 instead?

**Review by Manason — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 1 above)
Comment (mockdot) [reply client/main.lua]: This is just my OCD to remove the yellow line on IDE. I'll remove it.

File: client/main.lua (hunk 2 above)
Comment (mockdot) [reply client/main.lua:424]: Yes, the secondsLeft timer count is upwards, not down, so if you use the suggested condition your car will immediately despawn. I changed the number as it feels to me like a little network delay on some users (since it doesn't happen in any of my two local computers), but I do see it on some users that connect remotely.

**Review by Manason — APPROVED** [review]
